schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+  id INT GENERATED ALWAYS AS IDENTITY,
```

Comment:
# Optional:
 - [ ]You can make the id column not null and autoincremented: `id int NOT NULL IDENTITY(1,1)`